src/test/java/nextstep/subway/line/LineAcceptanceTest.java

```diff
@@ -0,0 +1,256 @@
+package nextstep.subway.line;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import nextstep.subway.Isolationer;
+import nextstep.subway.dto.LineDto;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.MediaType;
+
+import java.util.List;
+
+import static nextstep.subway.common.Common.지하철역을_생성한다;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertAll;
+
+@DisplayName("노선 관련 기능")
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class LineAcceptanceTest {
+
+    @LocalServerPort
+    int port;
+
+    @Autowired
+    private Isolationer dbIsolation;
+
+    @BeforeEach
+    public void setUp() {
+        if (RestAssured.port == RestAssured.UNDEFINED_PORT) {
+            RestAssured.port = port;
+        }
+        dbIsolation.excute();
+    }
+
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철 노선을 생성한다.")
+    @Test
+    void createLine() {
+        // given
+        int upStationId = 지하철역을_생성한다("강남역").jsonPath().get("id");
+        int downStationId = 지하철역을_생성한다("양재역").jsonPath().get("id");
+        String lineName = "신분당선";
+        LineDto.CreateRequest lineCreateRequest = new LineDto.CreateRequest()
+                .setName(lineName)
+                .setColor("bg-red-600")
+                .setUpStationId(upStationId)
+                .setDownStationId(downStationId)
+                .setDistance(10);
```

Comment:
```suggestion
        LineDto.CreateRequest 라인요청파라미터 = 라인_생성_요청파라미터(lineName...);
```

이렇게 한다면 좀 더 테스트코드에 집중하기 좋을 것 같아요